backend/src/main/java/gov/cdc/usds/simplereport/utils/UnknownAddressUtils.java

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.usds.simplereport.utils;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
+import java.util.Arrays;
+import org.apache.commons.lang3.StringUtils;
+
+public class UnknownAddressUtils {
+  private UnknownAddressUtils() {}
+
+  public static final String ADDRESS_STATE_UNKNOWN = "NA";
+  public static final String ADDRESS_ZIP_UNKNOWN = "00000";
+  public static final String ADDRESS_STREET_UNKNOWN = "** Unknown / Not Given **";
+
+  private static Boolean isAddressSectionUnk(String userAddressInput, String actual) {
```

Comment:
Can you make these return `boolean` primitive instead of the Object since you only need to know if it's true or false? 
Honestly the only time you really need to use the Object is in a place where it could be null like an optional GQL or REST parameter.

Reply:
OOO ok! got it! will do!